notebooks/Issue322/Issue322.ipynb

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
This notebooks they have to go away. Adding an example with the functionality would be great but it would need to be a .py file in the examples (new in GemPy 2.2)